standards/python.md

```diff
@@ -31,39 +32,50 @@ See [Imports in PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and [Im
 
 - Multiple-line `from` imports become grouped within parenthesis with one imported name per-line and a trailing comma for the last imported name.
-- Consider `six` to be equivalent to `__future__` imports.
 - Django imports are included as their own section between the standard library and third-party imports.
 - For Wagtail-specific projects like [cfgov-refresh](https://github.com/cfpb/cfgov-refresh), Wagtail imports are included as their own section between Django and third-party imports.
 
 The isort configuration can go into a `tox.ini` file under an `[isort]` header.
 
-## Python version
+## Dependency support/testing matrix
 
-We test against Python 3.6.
+We have two general types of Python projects:
 
-We recommend using [tox](https://tox.readthedocs.io/en/latest/) for matrix testing, and provide a [sample tox configuration](../tox.ini) that tests against Python 3.6 and Django LTS versions 1.8 and 1.11.
+- Applications that have a running instance, i.e. [cfgov-refresh](https://github.com/cfpb/cfgov-refresh)
+- Libraries that provide some functionality independent of applications
 
-For writing code that is Python 3-compatible, please see the [Python Porting HOWTO](https://docs.python.org/3/howto/pyporting.html) and the [Django Porting to Python3](https://docs.djangoproject.com/en/1.11/topics/python3/). In particular, we prefer to:
+When deciding what versions of Python and major dependencies they should support and be tested against, we use the following rubric:
 
-- [Use feature detection instead of version detection](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)
-- Use absolute imports (`from __future__ import absolute_import`)
-- Use the `print()` function instead of the `print` statement (`from __future__ import print_function`)
-- Use unicode literals (`from __future__ import unicode_literals`), and [the corresponding Django `python_2_unicode_compatible`](https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.encoding.python_2_unicode_compatible) (`from django.utils.encoding import python_2_unicode_compatible`)
-- Use Python 3 division wherein dividing `int`s results in a `float` (`from __future__ import division`)
-- Capture exceptions using the `as` keyword (`except Exception as exc`)
+|| Library | Application |
+|--|--|--|
+|Oldest Python 3 supported by major dependencies | ✔️| |
+|Current production Python | ✔️ | ✔️ |
+|Latest Python | ✔️ | ✔️ |
+|Current LTS version of major dependency | ✔️ | ✔️ |
+|Latest version of major dependency |  ✔️ | ✔️ |
 
-## `requirements.txt`, `setup.py`, and specifying dependencies
+Our applications should support the version of Python 3 we currently run in production and the latest version. For example, [cfgov-refresh](https://github.com/cfpb/cfgov-refresh) should support version 3.6 and 3.8.
```

Comment:
Consider adding "at time of writing", or something to make it so this sentence won't immediately go out of date.

Reply:
Done!